OSSMDOC-303: Config reorg (#32055)
openshift/openshift-docs: jaeger/jaeger_install/rhbjaeger-deploying.adoc

[NOTE]
====

> There are two ways to install and use Jaeger, as part of a service mesh or as a stand alone component. If you have installed Jaeger as part of Red Hat OpenShift Service Mesh, you can configure and deploy Jaeger as part of the xref:../../service_mesh/v2x/installing-ossm.adoc#installing-ossm[ServiceMeshControlPlane] or configure Jaeger and then xref:../../service_mesh/v2x/ossm-observability.html#ossm-config-external-jaeger_observability[reference your Jaeger configuration in the ServiceMeshControlPlane].

Reviewer: ```suggestion
If you installed Jaeger as part of Red Hat OpenShift Service Mesh, not as a standalone component, you have two options:

* You can configure and deploy Jaeger as part of the xref:../../service_mesh/v2x/installing-ossm.adoc#installing-ossm[ServiceMeshControlPlane]. 
* You can configure Jaeger and then xref:../../service_mesh/v2x/ossm-observability.html#ossm-config-external-jaeger_observability[reference your Jaeger configuration in the ServiceMeshControlPlane].

```

Author: This is good feedback, but it's in the standalone Jaeger docs. I made the change to make the link work. I feel like fixing that is out of scope.
